src/ocl/rnn_util_ocl.cpp

```diff
@@ -77,6 +88,8 @@ void LSTMForwardHiddenStateUpdate(const Handle& handle,
         auto kernel = kernels.front();
         kernel(cx,
                reserve_space,
+               static_cast<int>(hy_h),
```

Comment:
These vars are already of type int, isn't it?

Reply:
Yes, good catch.